pkg/network/state_test.go

```diff
@@ -170,12 +170,11 @@ func TestRetrieveClosedConnection(t *testing.T) {
 	t.Run("with registration", func(t *testing.T) {
```

Comment:
@hmahmood @brycekahle What you are referring to is already tested above this test : `without prior registration` and `with registration`.

Reply:
I guess this is the most we can test anyways, as I changed the whole "first call without registration" code path anyways, so we were relying on the stats reset, which does not exist anymore.

Reply:
@paulsemel could you add a test for "without prior registration" that ensures it gets the correct list of active connections?